### ISSUE TITLE
Fixed typos for results of OSNet-AIN on msmt->market/duke (table 9 in the paper)

### ISSUE THREAD

**Issue by ZhengRui** [KaiyangZhou/deep-person-reid#325]
Thanks for the great work and clean codebase. I tested the pretrained model (last row of Model Zoo) osnet_ain_x1_0 msmt17 -> market1501, I got same top-1 recall, but the map is 42.7 instead of 52.7, and by comparison with other rows, I think 42.7 is the right number.

**Comment by KaiyangZhou** [comment]
what's your full command used for evaluation? please make sure all setups follow the document (e.g. metric=cosine)

**Comment by ZhengRui** [comment]
42.7 comes from the test using my own codebase (i am able to get same performance for other pre-trained models except this one). I also tested using this repo and get 43.3 map with 70.1 top-1 recall, the command i use is:

`python scripts/main.py --config-file configs/im_osnet_ain_x1_0_softmax_256x128_amsgrad_cosine.yaml -t market1501 --root ./reid-data model.load_weights ./osnet_ain_x1_0_msmt17_256x128_amsgrad_ep50_lr0.0015_coslr_b64_fb10_softmax_labsmth_flip_jitter.pth test.evaluate True`

with result:
![image](https://user-images.githubusercontent.com/4105014/78033546-d542ba80-7366-11ea-9f3e-587b2741251f.png)

**Comment by KaiyangZhou** [comment]
this is what I got by testing the model on market and duke
```
##### Evaluating market1501 (source) #####
Extracting features from query set ...
Done, obtained 3368-by-512 matrix
Extracting features from gallery set ...
Done, obtained 15913-by-512 matrix
Speed: 0.1155 sec/batch
Computing distance matrix with metric=cosine ...
Computing CMC and mAP ...
** Results **
mAP: 43.3%
CMC curve
Rank-1  : 70.1%
Rank-5  : 84.1%
Rank-10 : 88.6%
Rank-20 : 91.9%
##### Evaluating dukemtmcreid (target) #####
Extracting features from query set ...
Done, obtained 2228-by-512 matrix
Extracting features from gallery set ...
Done, obtained 17661-by-512 matrix
Speed: 0.0856 sec/batch
Computing distance matrix with metric=cosine ...
Computing CMC and mAP ...
** Results **
mAP: 52.7%
CMC curve
Rank-1  : 71.1%
Rank-5  : 83.3%
Rank-10 : 86.4%
Rank-20 : 88.8%
```

it looks like I mistakenly swapped the numbers for market and duke, so R1=70.1 is for market and R1=71.1 is for duke

I will fix this
